test/spec/modules/eids_spec.js

```diff
@@ -237,8 +237,42 @@ describe('eids array generation for known sub-modules', function() {
       }]
     });
   });
+  it('PubProvided', function() {
+    const userId = {
+      pubProvided: [{
+        source: 'example.com',
+        type: 'pubProvided',
```

Comment:
this seems like a good alternative to using a generic `source` field; even though it's not a "standard" i don't think there's an issue with this for now. Probably makes sense to make sure this is included in the examples and documentation, though, since it's not mentioned anywhere but in your _spec files that you could use this.

Reply:
I will make sure that it will be in the documentation

Reply:
any chance we could take a stab at defining a few conventions in this doc? eg sha256email etc?